xml/docker_dockerizing_application.xml

```diff
@@ -10,35 +10,34 @@
  xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>Creating &docker; Images of Applications</title>
  <para>
-  &deng; is a technology that can help minimize resources used to run or
-  build applications. There are several types of applications that are
-  suitable to run inside a &docker; container like daemons, Web pages or
-  applications that expose ports for communication. You can use &deng; to
-  automate building and deployment processes by adding the build process
-  into a &docker; image, then building the image and then running containers
-  based on that image.
+  &deng; is a technology that can help minimize the resources used to run
+  applications. Several types of applications are suitable to be run inside
+  &docker; containers, such as daemons, Web pages or applications that expose
+  IP ports for communications. You can use &deng; to automate the building and
```

Comment:
quite a long sentence....i lost attention at its end :-)

Reply:
OK, it is a bit... but slightly shorter than it was before...?

Reply:
yes, it's better now